flixel/group/FlxSpriteGroup.hx

```diff
@@ -280,6 +281,9 @@ class FlxTypedSpriteGroup<T:FlxSprite> extends FlxSprite
 		sprite.alpha *= alpha;
 		sprite.scrollFactor.copyFrom(scrollFactor);
 		sprite.cameras = _cameras; // _cameras instead of cameras because get_cameras() will not return null
+		
+		if (clipRect != null) clipRectTransform(sprite, clipRect);
+		else sprite.clipRect = null;
```

Comment:
This seems to make this a breaking change, because it removes the `clipRect` that sprite might have had before being added. Not sure that's a good idea?